lisa/base_tools/service.py

```diff
@@ -116,6 +125,23 @@ def enable_service(self, name: str) -> None:
     def hibernate(self) -> None:
         self.run_async("hibernate", sudo=True, force_run=True)
 
+    def state(self) -> str:
+        cmd_result = self.run(
+            "status --no-page",
+            shell=True,
+            sudo=True,
+            force_run=True,
+        )
+        if cmd_result.exit_code != 0 and self.__NOT_FOUND.findall(cmd_result.stdout):
+            raise UnsupportedDistroException(
+                os=self.node.os, message=f"command not find {self.command}"
+            )
+        group = find_patterns_groups_in_lines(
```

Comment:
if there is only one group, try `find_patterns_group_in_lines`. It doesn't need to use 2-dim resutls.

Reply:
updated